oapen-engine/src/data/connection.py

```diff
@@ -20,6 +20,12 @@ def get_connection():
         db_version = cur.fetchone()
         print(db_version)
 
+        # psycopg2.extensions.register_type(
```

Comment:
Not sure if we should leave this in the final PR. If it's not something meant to be used by everyone and uncommented, I think you can get rid of it, but if it is, feel free to disregard this comment